cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -103,6 +103,12 @@ configure_file(
   @ONLY
 )
 
+
+set(pass_boost_no_cmake_config )
+if (@HUNTER_Boost_VERSION@ VERSION_GREATER_EQUAL 1.72.0)
```

Comment:
Will this work when the version has a suffix like `-p0`? If tests are passing, I guess so?

Reply:
Yes it does work with -p0 it ends capturing version numbers as soon as something else than . and numbers are found. So the -p0 just get ignored.

Reply:
Sweet